packages/api/lambdas/publish-reports.js

```diff
@@ -0,0 +1,152 @@
+'use strict';
+
+const merge = require('lodash.merge');
+
+const aws = require('@cumulus/common/aws');
+const {
+  getSfEventMessageObject,
+  getSfEventStatus,
+  isFailedSfStatus,
+  isTerminalSfStatus
+} = require('@cumulus/common/cloudwatch-event');
+// const {
+//   getMessageExecutionArn
+// } = require('@cumulus/common/message');
+const log = require('@cumulus/common/log');
+// const StepFunctions = require('@cumulus/common/StepFunctions');
+
+/**
+ * Publish a message to an SNS topic.
+ *
+ * Catch any thrown errors and log them.
+ *
+ * @param {string} snsTopicArn - SNS topic ARN
+ * @param {Object} eventMessage - Workflow execution message
+ * @returns {Promise}
+ */
+async function publishSnsMessage(
+  snsTopicArn,
+  eventMessage
+) {
+  try {
+    if (!snsTopicArn) {
+      throw new Error('Missing SNS topic ARN');
+    }
+
+    await aws.sns().publish({
+      TopicArn: snsTopicArn,
+      Message: JSON.stringify(eventMessage)
```

Comment:
As we discussed, it would be better if the topics received processed messages, instead of the full Cumulus message.